lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -115,26 +123,42 @@ $-separator-color: var(--nx-swatch-indigo-40);
     margin: 0;
   }
 
-  &:focus {
-    border-radius: 0;
-    outline: 2px solid $-focus-color;
-    outline-offset: -2px;
-  }
+  &.nx-text-link {
+    &:focus:not(.selected) {
```

Comment:
Rather than using `:not` here could the `&.selected` below just add `outline: none`?

Reply:
> Rather than using `:not` here could the `&.selected` below just add `outline: none`?

I'm not quite sure if I follow. The selected items needs to have an outline when focused. But I think this change makes more sense:
13233b2dfca85e37f79b427515349bfa1caa2c48